packages/server/graphql/mutations/updateCreditCard.ts

```diff
@@ -36,7 +36,7 @@ export default {
 
     // RESOLUTION
     const viewer = await dataLoader.get('users').load(viewerId)
-    await upgradeToPro(orgId, stripeToken, viewer.email)
+    await upgradeToPro(orgId, stripeToken, viewer!.email)
```

Comment:
I would prefer to have checks in these cases. As theoretically, it is not guaranteed that the viewer would exist, even if there is a small possibility. 
But not sure, we can try to formalize rules where `!` is acceptable. 
What if we disable `!` at all?
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-non-null-assertion.md